src/main/java/school/hei/haapi/endpoint/rest/mapper/RetakeExamMapper.java

```diff
@@ -33,21 +35,25 @@ public school.hei.haapi.model.RetakeExam toDomainCrupDate(CrupdateRetakeExam cru
         .student(studentUser)
         .session(retakeExamSession)
         .course(course)
+        .inscriptionDate(
+            Instant.parse(Objects.requireNonNull(crupdateRetakeExam.getInscriptionDate())))
         .build();
   }
 
   public RetakeExam toRest(school.hei.haapi.model.RetakeExam retakeExam) {
     return new RetakeExam()
         .id(retakeExam.getId())
         .course(courseMapper.toRest(retakeExam.getCourse()))
-        .session(retakeExamSessionMapper.toRest(retakeExam.getSession()));
+        .session(retakeExamSessionMapper.toRest(retakeExam.getSession()))
+        .inscriptionDate(String.valueOf(retakeExam.getInscriptionDate()));
```

Comment:
No. You use String here only because you map badly the format in the API spec. Instead of format: date-time, you wrote format: datetime. 

Actually, the appropriate type here, once you'll do the change, will directly be Instant type.